src/rt/osx_tls.c

```diff
@@ -1,51 +0,0 @@
-/**
- * Helpers for determining TLS memory ranges on OS X.
- *
- * This unfortunately cannot be entirely done in D, as the OS X API uses
- * the Apple-specific blocks C extension.
- *
- * Copyright: David Nadlinger, 2012.
- * License:   <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
- * Authors:   David Nadlinger
- */
-
-#ifndef __APPLE__
-#ifndef __BLOCKS__
- #error "Need a C compiler with Apple Blocks support – not building on OS X with Clang?"
-#endif
-#endif
```

Comment:
Is there an equivalent test needed in the D version? That is, what happens if you are building on something other than OSX, or linking with non-clang libc?

I totally admit, I'm not sure what should be done, I'm just pointing out a difference between the C code and the D code.